src/OrchardCore.Modules/OrchardCore.Lucene/Views/LuceneContentIndexSettings.Edit.cshtml

```diff
@@ -0,0 +1,24 @@
+@model OrchardCore.Lucene.Settings.LuceneContentIndexSettingsViewModel
+<h4>Lucene</h4>
```

Comment:
Here @sebastienros I added the "Lucene" title on the Content Type editor view. That's why I thought it was already there. But I'm adding visual a distinction in this PR